Redesign game menu

Heartfyre opened the pull request:
![gamemenu](https://user-images.githubusercontent.com/20871500/41819450-1395882e-77c1-11e8-8dda-bd3fa4f9f22d.PNG)

Ajackster commented:
Looks great, just a couple items. Talked to @codecorsair and @jamkoo 
1.) Can we have an X box instead of the "Close Menu" button?
2.) The title and button fonts are off, the Settings menu has the correct fonts so just copy those
3.) Can we make the widget a bit darker, similar to the darkness of the Settings menu? Attached is a quick mockup of the darkness

![kooscreen1529954557 png](https://user-images.githubusercontent.com/14223893/41871247-615cdef6-788c-11e8-97e8-7223d7a712a6.png)

Heartfyre commented:
![menu](https://user-images.githubusercontent.com/20871500/41980925-03db076a-7a28-11e8-95d8-9e73b1413e16.PNG)

Added a darker background, reworked the title graphics, so it fits better with the smaller window, reworked the buttons and the glow (they're bigger than the main ones), changed font-size and color, so it fits the other windows (settings). I also removed the close button from the footer, and the footer as well , and added the close button the the top right corner.

Ajackster commented on game/hud/src/components/GameMenu/index.tsx:
```diff
@@ -17,59 +17,120 @@ export interface GameMenuStyle {
-  border-top: 1px solid ${utils.lightenColor('#202020', 30)}
+const MenuCorner = styled('div')` {
+  position: absolute;
+  min-width: 320px;
```
Any reason you set absolute width and heights for these guys? Could just use 100% for the dimensions

Ajackster replied:
You don't need to update this, I can do on local comp if there's no problems

Ajackster commented:
I'm gnna merge this in and then make some small modifications. The modifications are
1.) A little less padding on the sides
2.) Reuse background image that MOTD is using

Thanks @Heartfyre !!
